lib/handlers/formatting.js

```diff
@@ -25,104 +25,86 @@
 import _ from 'underscore';
 
 import * as exec from '../exec';
+import * as all from '../formatters/_all';
```

Comment:
This is not the intended way to use the "_all" pattern as seen in other parts of the codebase.
There is meant to be a corresponding index.js for every _all.js, and _all.js should never be imported from outside that index.js

See the demanglers for an example: https://github.com/compiler-explorer/compiler-explorer/blob/main/lib/demangler/index.js

Reply:
Fixed in 75e744eeaea38728734aa152002588fe9fa4093d